backstop/tests/iModel-loader.html

```diff
@@ -0,0 +1,413 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>iModel loader | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      .iui-progress-indicator-full-page {
+        height: 90vh;
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .iui-loading-imodel {
+        width: 250px;
+        margin-bottom: 20px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <section>
+      <div
+        id="imodel-loader"
+        class="iui-progress-indicator-full-page"
+      >
+        <!-- Use iModel-loader.svg from @itwin/itwinui-illustrations package. -->
+        <svg
+          class="iui-loading-imodel"
```

Comment:
as this class is not in illustartion itself (is it?), maybe lets not use this class (with iui prefix) and instead have selector using `imodel-loader` id and taclking svg with what ever rules you need